server/src/routes/api/v1/boardGamesRouter.js

```diff
@@ -44,4 +45,17 @@ boardGamesRouter.get("/:id", async (req, res) =>{
 	}
 })
 
+boardGamesRouter.delete("/:id", async (req, res) =>{
+	const { id } = req.params
+	try {
+		const deletedBoardGame = await BoardGame.query().delete().where("id", id)
+		if (deletedBoardGame) {
```

Comment:
this is sort of okay, but if a board game is deleted, `delete()` returns the number of rows deleted (in this case "1")
it would be better to check if `deletedBoardGame` is equal to 1

and you could rename `deletedBoardGame` to `rowsDeleted`